### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by no4kar**
- [DEMO LINK](https://no4kar.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://no4kar.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,103 @@
 body {
   margin: 0;
+  --link-hover-color: #00acdc;
+  --link-active-color: var(--link-hover-color);
+  font-family: "Roboto", sans-serif;
```
Comment (Esceype) [inline src/style.css:5]: Checklist 3

File: src/index.html [hunk 2]
```diff
@@ -3,12 +3,66 @@
...
+          </li>
+          <li class="nav__item">
+            <a href="#home" class="nav__link">Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#home" class="nav__link">Photo
+            </a>
+          </li>
+          <li class="nav__item">
```
Comment (Esceype) [inline src/index.html:57]: Empty line here and in all similar cases

File: src/index.html [hunk 3]
```diff
@@ -3,12 +3,66 @@
...
+            <a href="#home" class="nav__link">Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#home" class="nav__link">Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#home" class="nav__link"
+            data-qa="hover">Laptops & Computers
```
Comment (Esceype) [inline src/index.html:42]: Checklist 13

File: src/index.html [hunk 4]
```diff
@@ -3,12 +3,66 @@
...
   </head>
-  <body>
-    <h1>Moyo header</h1>
+
+  <body id="home">
+
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+        src="https://s3-alpha-sig.figma.com/img/833f/9aab/47d660efd384318772b6b8040db831aa?Expires=1690761600&Signature=Q3cqcj6MIMPNPsJsRjDOP7bAZG~6TE32Qn-SER0QE7Da0XP19DN6kVk50DmsEL1jpt-Ohg0HB5StWjnhzcgsoHZ7oax8ypiRCkQcskqzpErSqKpx6pvOOGgWCfMadYxjGSE8jLo3VjxP6BXqTg8WW5xN3KSHvXnuS2HVd0MM5uPx-VrFU-dGjQNkVAX8HkRMrNJn1z2Acwy2GE6m8Ifn8I58PH5GcuHLwXqsYzFekoaeR5ebrIU-XxRtA9VwpNOgSa6wWUO08D0fE6BTlmIPDF~7JOSUCumW0hiz9WhUxgcLnW57ekqc~6VUPGIfQjJkrAeFbhhIgXO2uui9AfMm9w__&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4"
```
Comment (Esceype) [inline src/index.html:21]: Requirements point number 3

File: src/index.html [hunk 5]
```diff
@@ -3,12 +3,66 @@
...
+        alt="MOYO.png"
```
Comment (Esceype) [inline src/index.html:22]: Bad alt naming

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,103 @@
...
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+}
+
+.nav__item {
+  /* padding: 0 22px 0 23px 0; */
+  /* border: 1px solid red; */
+}
```
Comment (Esceype) [inline src/style.css:50]: Remove comments like that if you finish your work

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,103 @@
...
+  left: 0;
+  border-radius: 8px;
+}
+
+.nav__item:first-child .nav__link {
+  margin-left: 0;
+  /* border: 1px solid red; */
+}
+
+a.is-active {
```
Comment (Esceype) [inline src/style.css:90]: Checklist 1

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,103 @@
...
+  position: relative;
+  right: 50px;
+  display: flex;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
```
Comment (Esceype) [inline src/style.css:41]: It not necessary cause row direction is default for flex box

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,103 @@
...
+  list-style: none;
+}
+
+.nav__item {
+  /* padding: 0 22px 0 23px 0; */
+  /* border: 1px solid red; */
+}
+
+.nav__link {
+  display: flex;
```
Comment (Esceype) [inline src/style.css:53]: For what do you provide this CSS property in every selector ?

File: src/style.css [hunk 10]
```diff
@@ -1,3 +1,103 @@
...
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 1200px;
+  min-width: 1024px;
+  margin: 0 auto;
+  background-color: #fff;
```
Comment (Esceype) [inline src/style.css:15]: Move it to CSS variable

File: src/style.css [hunk 11]
```diff
@@ -1,3 +1,103 @@
...
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  margin: 0;
+  position: relative;
+  right: 50px;
+  display: flex;
+}
```
Comment (Esceype) [inline src/style.css:35]: Try to make it without using position

File: src/index.html [hunk 12]
```diff
@@ -3,12 +3,76 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+
+  <body id="home">
+
+    <header class="header">
```
Comment (polosanya) [inline src/index.html:18]:
```suggestion
  <body id="home">
    <header class="header">
```
Don't leave empty lines between parent and child elements

File: src/index.html [hunk 13]
```diff
@@ -3,12 +3,76 @@
...
+      <a href="#home" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="logo-img.png"
+        class="logo__img">
```
Comment (polosanya) [inline src/index.html:23]:
```suggestion
        <img
          src="./images/logo.png"
          alt="logo-img.png"
          class="logo__img"
        >
```
Fix all cases
image

File: src/index.html [hunk 14]
```diff
@@ -3,12 +3,76 @@
...
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#home" class="nav__link is-active">Apple
+            </a>
```
Comment (polosanya) [inline src/index.html:30]:
```suggestion
            <a href="#home" class="nav__link is-active">
              Apple
            </a>
```
Fix all cases

File: src/index.html [hunk 15]
```diff
@@ -3,12 +3,76 @@
...
+            <a href="#home" class="nav__link">Video
+            </a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
   </body>
+
```
Comment (polosanya) [inline src/index.html:77]: Remove empty lines

File: src/style.css (hunk 9 above)
Comment (polosanya) [reply src/style.css:44]: Not fixed. Display flex is actually redundant here. Set the correct line-height to center it vertically

File: src/style.css [hunk 16]
```diff
@@ -1,3 +1,92 @@
...
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin-left: 20px;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  position: relative;
```
Comment (polosanya) [inline src/style.css:52]: Pay attention to CSS properties order
https://9elements.com/css-rule-order/

**Review by polosanya — CHANGES_REQUESTED** [review]
Almost done!

File: src/style.css [hunk 17]
```diff
@@ -1,3 +1,92 @@
...
+  width: 100%;
+  bottom: 0;
+  left: 0;
+  background-color: var(--link-active-color);
+  border-radius: 8px;
+}
+
+.nav__item:first-child .nav__link {
+  margin-left: 0;
+}
```
Comment (xiimiik) [inline src/style.css:77]: do not increase specifity
delete margin-left from `nav__link`  and  try instead
```
.nav__item:not(:first-child) {
  margin-left: 20px;
}
```

File: src/index.html [hunk 18]
```diff
@@ -3,12 +3,82 @@
...
+        class="logo__img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#home" class="nav__link is-active">
+              Apple
+            </a>
```
Comment (xiimiik) [inline src/index.html:31]: checklist point number '13 ❗

File: src/index.html [hunk 19]
```diff
@@ -3,12 +3,82 @@
...
-    <h1>Moyo header</h1>
+
+  <body id="home">
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="logo-img.png"
+        class="logo__img"
+        >
```
Comment (xiimiik) [inline src/index.html:23]: checklist point number '13 ❗

File: src/index.html [hunk 20]
```diff
@@ -3,12 +3,82 @@
...
+          </li>
+
+          <li class="nav__item">
+            <a
+            href="#home"
+            class="nav__link"
+            data-qa="hover"
+            >
+            Laptops & Computers
+            </a>
```
Comment (xiimiik) [inline src/index.html:53]: checklist point number '13 ❗

**Review by xiimiik — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 21]
```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
```
Comment (Esceype) [inline src/style.css:3]: GJ

File: src/style.css [hunk 22]
```diff
@@ -1,3 +1,74 @@
...
+}
+
+.logo__img {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  margin: 0 50px 0 0;
```
Comment (Esceype) [inline src/style.css:34]: For the future, you can`t determinate external position of BEM block, but you can do it inside BEM element of parent BEM block

**Review by Esceype — APPROVED** [review]
GJ !!! Read the comments for better understanding of BEM methodology

**Pull request closed without merging**